server/grpc/gogoreflection/fix_registration.go

```diff
@@ -0,0 +1,143 @@
+package gogoreflection
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"reflect"
+
+	_ "github.com/gogo/protobuf/gogoproto" // required so it does register the gogoproto file descriptor
+	gogoproto "github.com/gogo/protobuf/proto"
+
+	// nolint: staticcheck
+	"github.com/golang/protobuf/proto"
+	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
```

Comment:
why creating an alias?

Reply:
what do you mean, i didn't catch it

Reply:
if you mean dpb, it's grpc-reflection imports this package, so keeping things consistent

Reply:
The alias is `dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"`
I think Robert was asking why you use an alias when there doesn't seem to any conflicts in the file's namespace.

Reply:
the grpc reflection package imports the package `"github.com/golang/protobuf/protoc-gen-go/descriptor"` as `dpb`. So I've done it in the same way to keep things consistent

Reply:
Normally they should be used only for solving import conflicts.